README.md

```diff
@@ -98,10 +98,11 @@ calling etc.*
 Antonis Tsipaliokas • [GitHub](https://github.com/kokeroulis)  
 Daniel Sobczak • [GitHub](https://github.com/DanielSobczak) (remote)  
 
-##### Partnerships squad
+##### Digital first primary care squad
 
-*Working on our partners applications*
+*Working on our and our partners applications (Babylon GP at Hand, Babylon by TelusHealth)*
```

Comment:
> Working on our and our partners

Not quite good english

Reply:
_Working on our partners applications and our own_

That would work? :D

Reply:
Open to suggestions 😄

Reply:
"Working on our own and our partners' applications" might be marginally better

or even

"Working on our and our partners' applications". The only different from your spelling was including the apostrophe. I _think_ it should be there (but I'm not sure). I think it partially confused me because I was reading it quickly so that's not your fault, but the apostrophe makes it more obvious there is a possessive there which helps me comprehending the sentence.